datafusion-examples/examples/data_io/parquet_encrypted.rs

```diff
@@ -32,13 +33,14 @@ pub async fn parquet_encrypted() -> datafusion::common::Result<()> {
     // The SessionContext is the main high level API for interacting with DataFusion
     let ctx = SessionContext::new();
 
-    // Find the local path of "alltypes_plain.parquet"
-    let testdata = datafusion::test_util::parquet_test_data();
-    let filename = &format!("{testdata}/alltypes_plain.parquet");
+    let path = PathBuf::from(env!("CARGO_MANIFEST_DIR"))
```

Comment:
Maybe we can change this example to read in the cars.csv file and then write it back out as an encrypted parquet file?